archinfo/arch_ppc32.py

```diff
@@ -1,12 +1,21 @@
+import logging
+
+l = logging.getLogger("archinfo.arch_ppc32")
+
 try:
     import capstone as _capstone
 except ImportError:
     _capstone = None
 
+try:
+    import keystone as _keystone
+except ImportError:
+    _keystone = None
+
 #try:
-#   import unicorn as _unicorn
+#    import unicorn as _unicorn
 #except ImportError:
-#   _unicorn = None
+#    _unicorn = None
```

Comment:
Same as above - this change is not necessary.

Reply:
Bizarre... I don't see those extra spaces at all?

Reply:
That line should not be changed. Are you saying you don't see those extra spaces in your local editor? Maybe there is some special setting with your editor?

Reply:
I know why, the extra spaces are correct since we have '#' at the start of these lines (I assume the correct tab width is 4)